src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -26,7 +27,7 @@ const styles = StyleSheet.create({
 });
 
 const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
-  const {selectedChannel, handleUserName} = useChatScreenHook(type || ANONYMOUS);
+  const {selectedChannel} = useChatUtilsHook(type || ANONYMOUS);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `selectedChannel` is now being fetched from `useChatUtilsHook` instead of `useChatScreenHook`. Make sure that `useChatUtilsHook` provides the same functionality as the previous hook.
```diff
-  const {selectedChannel, handleUserName} = useChatScreenHook(type || ANONYMOUS);
+  const {selectedChannel} = useChatUtilsHook(type || ANONYMOUS);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->